tools/lib/h5tools_utils.c

```diff
@@ -194,44 +194,50 @@ get_option(int argc, const char **argv, const char *opts, const struct long_opti
 
     if (sp == 1 && argv[opt_ind][0] == '-' && argv[opt_ind][1] == '-') {
         /* long command line option */
-        const char *arg = &argv[opt_ind][2];
         int         i;
```

Comment:
Use actual long name from array to long name option supplied on command line for compare.